src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py

```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
 
     :example: Create a ticket to request an increase in DTUs for Azure Synapse Analytics
-        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/sql_datawarehouse_problemClassification_guid" --severity "moderate" --ticket-name "SynapseAnalyticsDTUsQuotaIncreaseRequest" --title "Request for DTUs Quota Increase in Azure Synapse Analytics"
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/machine_learning_service_problemClassification_guid" --severity "moderate" --ticket-name "SynapseAnalyticsDTUsQuotaIncreaseRequest" --title "Request for DTUs Quota Increase in Azure Synapse Analytics"
 
     :example: Create a ticket to request an increase in DTUs for SQL Database
-        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/sql_database_problemClassification_guid" --severity "moderate" --ticket-name "SQLDatabaseDTUsQuotaIncreaseRequest" --title "Request for DTUs Quota Increase in SQL Database" --quota-change-version "1.0" --quota-change-request-sub-type "DTUs" --quota-change-requests '[{"region":"EastUS","payload":"{"ServerName":"testserver","NewLimit":54000}"}]'
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/sql_database_problemClassification_guid" --severity "moderate" --ticket-name "SQLDatabaseDTUsQuotaIncreaseRequest" --title "Request for DTUs Quota Increase in SQL Database" --quota-change-version "1.0" --quota-change-request-sub-type "DTUs" --quota-change-requests '[{"region":"EastUS","payload":"{"ServerName":"testserver","NewLimit":54000}"}]'

```

Comment:
--quota-change-requests '[{"region":"EastUS","payload":"{"ServerName":"testserver","NewLimit":54000}"}]' this is not what we had agreed upon, please update to the expected value